atm.css

```diff
@@ -18,7 +18,7 @@ body {
   font-size: 50px;
 }
 
-.account {
+.checking_account, .savings_account {
```

Comment:
Since this is common to both, it may make sense to leave the "account" class, but add an additional class (or ID) for specificity in other contexts.